File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/EnemyOceanDragonKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.DragonType;

import static com.bonepl.chromaleague.hud.PredefinedKeySets.BLACKWIDOW_FUNCTIONAL;

public class EnemyOceanDragonKillAnimation extends StaticPartialBlinkingAnimation {
    public EnemyOceanDragonKillAnimation() {
        super(BLACKWIDOW_FUNCTIONAL, 8, DragonType.OCEAN.getColor());
    }
}

File: src/hud/animations/EnemyOceanDragonKillAnimation.ts
import {DragonType} from '../../rest/eventdata/DragonType.js';
import {EnemyDragonKillBlinkingAnimation} from './EnemyDragonKillBlinkingAnimation.js';

export class EnemyOceanDragonKillAnimation extends EnemyDragonKillBlinkingAnimation {
    constructor() {
        super(DragonType.OCEAN);
    }
}
